internal/cmd/primaryip/list.go

```diff
@@ -38,6 +38,13 @@ var ListCmd = base.ListCmd{
 	OutputTable: func(client hcapi2.Client) *output.Table {
 		return output.NewTable().
 			AddAllowedFields(hcloud.PrimaryIP{}).
+			AddFieldFn("ip", output.FieldFn(func(obj interface{}) string {
+				primaryIP := obj.(*hcloud.PrimaryIP)
+				if primaryIP.Network != nil {
+					return primaryIP.Network.String()
+				}
```

Comment:
```suggestion
				// .Network is set when .Type == "ipv6"
				if primaryIP.Network != nil {
					return primaryIP.Network.String()
				}
```

This seems to hide some logic, and doesn't make the code obvious. Feel free to add my comment (maybe also in the floating IP implementation?) if you think it makes it more understandable.

Maybe having a `.Type == "ipv6"` condition instead of `.Network != nil` would be more explicit, but lets not diverges from the original floating IP implementation.

Reply:
Yes, I think a comment would be useful. I'll add that. I don't think changing ``.Network != nil`` to ``.Type == "ipv6"`` would be good idea though because that would make a nil dereference possible.

Reply:
Yes agreed.